view/frontend/web/css/styles.css

```diff
@@ -206,19 +206,23 @@
 /* General styling for checkout */
 
 .payment-method-content .field input, .payment-method-content .field select {
-    max-width: 168px;
+    max-width: 250px;
 }
 
 .adyen-checkout__link adyen-checkout__link--more-information {
     margin-bottom: 10px;
 }
 
+.checkout-component-dock {
+    max-width: 250px;
+    margin-bottom: 10px;
+}
+
```

Comment:
Shouldn't this be responsibility of the merchant to design?

Reply:
Yes it is, but we usually make it just a bit nicer so merchants still get a nice out of the box solution

Reply:
But it also makes it harder for them to customize it, no?